apps/judicial-system/api/src/app/data-sources/backend.ts

```diff
@@ -240,6 +240,10 @@ export class BackendApi extends DataSource<{ req: Request }> {
   ): Promise<DeleteDefendantResponse> {
     return this.delete(`case/${caseId}/defendant/${defendantId}`)
   }
+
+  getRestrictedCase(id: string): Promise<Case> {
```

Comment:
Good observation. However, I didn't want to limit this to defenders as I saw other uses for this endpoint, for instance for defendants or victims. Perhaps it would be better to call this a limited case, partial case or something to that effect. Semantically, this is a case which does not include all the case properties. I'd like to think about this a bit and have created an Asana ticket for the cool down week: https://app.asana.com/0/1199153462262248/1202211680145826/f

Reply:
Sounds good 👍